samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTime.Parse/vb/Parse2.vb

```diff
@@ -4,15 +4,15 @@ Option Strict On
 ' <Snippet2>
 Module Example
    Public Sub Main()
-      Dim dateStrings() As String = {"2008-05-01T07:34:42-5:00", _
-                                     "2008-05-01 7:34:42Z", _
+      Dim dateStrings() As String = {"2008-05-01T07:34:42-5:00", 
+                                     "2008-05-01 7:34:42Z", 
                                      "Thu, 01 May 2008 07:34:42 GMT"}
       
-      For Each dateString As String In dateStrings
+      For Each dateString In dateStrings
          Dim convertedDate As Date = Date.Parse(dateString)
-         Console.WriteLine("Converted {0} to {1} time {2}", _
-                           dateString, _
-                           convertedDate.Kind.ToString(), _
+         Console.WriteLine("Converted {0} to {1} time {2}", 
```

Comment:
I'd use the interpolated string syntax here. It would be more clear for the reader.